modules/contenta_graphql/contenta_graphql.module

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_entity_bundle_create().
+ */
+function contenta_graphql_entity_bundle_create($entity_type_id, $bundle) {
```

Comment:
@e0ipso what do you think about this approach to expose newly created content models to grapghQL?

Reply:
This can work. However I would recommend checking the GraphQL module PR queue. I think they have a solution there ready for review.

Maybe we can import that here?

Reply:
We are already exposing all configured fields (the raw typed data values) by default by now (https://github.com/fubhy/graphql-drupal/issues/335) and the same is planned for base fields (https://github.com/fubhy/graphql-drupal/issues/343). At that point you should be able to get all information out of the box.

Configuration is only necessary if you want special behaviours like rendered fields or images. There are ideas on how to make them easier accessible too, but no concrete plans yet.